.github/workflows/stale.yml

```diff
@@ -25,3 +25,5 @@ jobs:
           days-before-issue-stale: -1
           days-before-pr-close: 7
           days-before-issue-close: -1
+          exempt-draft-pr: true
```

Comment:
I do not want to manually close abadoned draft PRs. It is possible to reopen/recreate PR when it is needed.
``

```suggestion
```

Reply:
What's the underlying reason for this change? 
Is there a recent problem that you're trying to solve?

I think exempting drafts would be problematic as it now becomes a manual task to watch and close. 
It currently requires 14 days of idle activity before any PR is closed.

I'm not sure that "proposals" is a good enough justification. Users can use Issues for proposals which don't expire as quickly. Code samples can be referenced in a draft pr even after it's been closed. And as @Kielek already mentioned, it's possible to reopen a PR when needed.